client/clientservice/src/request_service.cpp

```diff
@@ -46,16 +46,16 @@ Status RequestServiceImpl::Send(ServerContext* context, const Request* proto_req
     if (not std::holds_alternative<bft::client::Reply>(send_result)) {
       LOG_INFO(logger_, "Send returned error");
       switch (std::get<uint32_t>(send_result)) {
-        case (concord_client_pool::Overloaded):
+        case (bftEngine::OperationResult::EXEC_DATA_TOO_LARGE):
           status.set_value(grpc::Status(grpc::StatusCode::RESOURCE_EXHAUSTED, "All clients occupied"));
           break;
-        case (concord_client_pool::InvalidArgument):
+        case (bftEngine::OperationResult::INVALID_REQUEST):
           status.set_value(grpc::Status(grpc::StatusCode::INVALID_ARGUMENT, "Invalid argument"));
           break;
-        case (concord_client_pool::TimedOut):
+        case (bftEngine::OperationResult::TIMEOUT):
           status.set_value(grpc::Status(grpc::StatusCode::DEADLINE_EXCEEDED, "Timeout"));
           break;
-        case (concord_client_pool::ClientUnavailable):
+        case (bftEngine::OperationResult::NOT_READY):
```

Comment:
In order to get it working, you should replace the set of ClientUnavailable value in the code with NOT_READY.